pkg/cmd/cli/cmd/status.go

```diff
@@ -160,7 +160,9 @@ func (o StatusOptions) RunStatus() error {
 		if err != nil {
 			return err
 		}
-		data, err := dot.Marshal(g, o.namespace, "", "  ", false)
+		// DOT graph names cannot contain dashes unless it is a double-quoted string
+		// See http://www.graphviz.org/doc/info/lang.html
+		data, err := dot.Marshal(g, fmt.Sprintf("%q", o.namespace), "", "  ", false)
```

Comment:
PR to change Marshal to quote when needed seems ok, and possibly special case an already-quoted name in case other consumers already worked around this the same way